### ISSUE TITLE
python 3.7 issues

### ISSUE THREAD

**Issue by brettowe** [20c/vodka#12]
while trying to use vaping I encountered an issue with vodka starting
looks to related to new 'async' built-in word in python 3.7
attached is a patch for this issue
I think there is other issues as I can't start vaping using the uwsgi or letting vaping run in background 
and the uwsgi error mentions that it can't find vodka

[vodka-python3.7.txt](https://github.com/20c/vodka/files/3000810/vodka-python3.7.txt)

**Comment by vegu** [comment]
@brettowe thanks for the report

**Comment by pmjdebruijn** [comment]
The attached vodka-python3.7.txt is actually a valid patch, would you please consider applying it and doing a new vodka release?

**Comment by vegu** [comment]
Sadly i haven't had time to look into vodka's issues with python 3.7 yet, the patch is appreciated, but as @brettowe said there seem to be additional issues. Will try to schedule some time to look at this this month.

@pmjdebruijn does that mean you applied the patch and vaping with vodka runs correctly on your py37 setup?

**Comment by hedronist** [comment]
I encountered this problem on a Raspberry Pi 3 running Raspbian 10 (buster) with Python 3.7. Affected files were:
```
.../python3.7/dist-packages/vodka/plugins/__init__.py (1 instance)
.../python3.7/dist-packages/vodka/__init__.py (4 instances)
```
In both files, renaming variables called 'async' to 'asynk' (or whatever) seems to fix the problem. I ran the ```examples/standalone_dns``` code and the results looked good in a browser aimed at it.

Almost certainly not all of the problems, but I'm just getting started.
